pkgs/development/python-modules/hebg/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+
+  # Runtime dependencies
+  networkx,
+  matplotlib,
+  numpy,
+  tqdm,
+  scipy,
+
+  # Build, dev and test dependencies
+  git,
+  setuptools-scm,
+  pytestCheckHook,
+  pytest-check,
+  pytest-mock,
+  sphinx,
+}:
+
+buildPythonPackage rec {
+  pname = "hebg";
+  version = "0.2.4";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-11bz+FbnaEVLiXT1eujMw8lvABlzVOeROOsdVgsyfjQ=";
+  };
+
+  build-system = [
+    git
+    setuptools-scm
+  ];
+
+  propagatedBuildInputs = [
+    networkx
+    matplotlib
+    numpy
+    tqdm
+    scipy
+  ];
+
```

Comment:
Could you add `pythonImportsCheck` for a smoke test?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck-using-pythonimportscheck

Reply:
I did for hebg, I'm not sure that is what you wanted tough